internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -322,7 +322,7 @@ func resourceLinuxVirtualMachine() *pluginsdk.Resource {
 			"secure_boot_enabled": {
 				Type:     pluginsdk.TypeBool,
 				Optional: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
`ForceNew` is false by default, so rather than setting it here, could we just remove it entirely?

```suggestion
```